### PR TITLE
add dark theme

### PR DESCRIPTION
**Pull request by AntoniiaK**

### CODE REVIEW

**Comment by AntoniiaK** [comment]
Ok, I'll do tests. Yes it's incorrect configuration of gitprofile. Thanks.

File: src/poliastro/tests/test_plotting3d.py [hunk 1]
```diff
@@ -88,6 +88,12 @@ def test_plot_trajectory_plots_a_trajectory():
     assert frame._attractor == Sun
 
 
+def test_dark_theme():
+    frame = OrbitPlotter3D(dark=True)
+    print(frame._layout.template.layout)
```
Comment (Juanlu001) [inline src/poliastro/tests/test_plotting3d.py]:
```suggestion
```

No need to put the `print` there, perhaps a remainer from the debugging?

**Comment by Juanlu001** [comment]
Also, there is one `PEP8` issue that appeared in the CircleCI logs:

https://circleci.com/gh/poliastro/poliastro/1052

```
src/poliastro/tests/test_plotting3d.py:120:1: W391 blank line at end of file
```

File: src/poliastro/tests/test_plotting3d.py [hunk 2]
```diff
@@ -111,3 +117,4 @@ def test_savefig_calls_prepare_plot(mock_prepare_plot, mock_export):
 
     assert mock_export.call_count == 1
     mock_prepare_plot.assert_called_once_with()
+
```
Comment (Juanlu001) [inline src/poliastro/tests/test_plotting3d.py]: No need for this extra line.

**Comment by AntoniiaK** [comment]
Thank you for your corrections.I will be more careful next time.

**Comment by Juanlu001** [comment]
@AntoniiaK no problem! I made a commit in your branch to fix another minor whitespace issue and now the tests are running. You can check right before committing with `git diff --cached`.

**Comment by Juanlu001** [comment]
The diff coverage is not high enough because there are no new tests for `plot` and `plot3d`, but I don't think it's that important, and also the overall coverage increased, so I'm merging.

**Comment by Juanlu001** [comment]
Thanks a lot for this first contribution :tada: Feel free to pick another issue!

**Pull request merged**